be/src/connector/mysql_connector.cpp

```diff
@@ -206,7 +206,7 @@ Status MySQLDataSource::get_next(RuntimeState* state, vectorized::ChunkPtr* chun
         return Status::EndOfFile("finished!");
     }
 
-    *chunk = vectorized::ChunkHelper::new_chunk(*_tuple_desc, state->chunk_size());
+    *chunk = vectorized::ChunkHelper::new_chunk(*_tuple_desc, 0);
```

Comment:
I think you don't need to change this value. if you allocate a 0 rows chunk, it can hold 0 rows.

Reply:
actually i think fill_chunk will append columns for the chunk. let me test later

Reply:
Consider to do some factor:
1. put `_tuple_desc` into `DataSource`
2. put `create default chunk` into `Data Source`
3. add a method `set default chunk size`in `Data Source`

So all `DataSource` can benefit this code.